modules/automatically-scaling-machines-to-available-bare-metal-hosts.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+[id="automatically-scaling-machines-to-available-bare-metal-hosts_{context}"]
+= Automatically scaling machines to the number of available bare metal hosts
+
+You can configure `Machine` objects to be automatically created to match the number of available `BareMetalHost` objects. This is done by adding a `metal3.io/autoscale-to-hosts` annotation to the `MachineSet` object that you want to be scaled automatically.
+
+.Prerequisites
+
+* Install {op-system} bare metal compute machines for use in the cluster, and create corresponding `BareMetalHost` objects.
+* Install the {product-title} CLI (oc).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Annotate the machine set that you want to configure for automatic scaling by adding the `metal3.io/autoscale-to-hosts` annotation. Replace `<machineset>` with the name of the machine set to be changed.
++
+[source,terminal]
+----
+$ oc annotate machineset <machineset> -n openshift-machine-api 'metal3.io/autoscale-to-hosts=<any_value>'
+----
++
+Wait for the new scaled machines to start.
+
+[NOTE]
+====
+When a `BareMetalHost` object has been used to create by a machine in the cluster and labels or selectors are subsequently changed on the `BareMetalHost`, the `BareMetalHost` object will continue be counted against the `MachineSet` that the `Machine` object was created from.
```

Comment:
Passive voice. has been used, are subsequently changed, continue to be counted, was created from.